packages/x-date-pickers-pro/src/DateRangePicker/DateRangePicker.test.tsx

```diff
@@ -9,7 +9,7 @@ import {
   openPicker,
   stubMatchMedia,
 } from 'test/utils/pickers';
-import { pickerPopperClasses } from '@mui/x-date-pickers/internals/components/PickerPopper';
+import { pickerPopperClasses } from '@mui/x-date-pickers/internals';
```

Comment:
This wasn't causing issues and, for some reason, didn't even trigger ESLint issue, even though it should have. 🤔